### ISSUE TITLE
After updating to 0.9.30 the plugin stopped working.

### ISSUE THREAD

**Issue by ivangolubev** [miximka/I2PMonitor#2]
As written in update description: 0.9.30 contains an upgrade to Jetty 9 and Tomcat 8. The previous versions are no longer supported

The error I see is:
Error starting plugin I2PControl: java.lang.Exception: Plugin requires Jetty version 8.9999 or lower

**Comment by miximka** [comment]
Yeah I saw it too... It would be great if the plugin maintainer would update it for the new Jetty.
